api/system-status/read

```diff
@@ -32,7 +32,48 @@ function readMemory()
     return $fields; 
 }
 
-function readUptime() {
+function readDisk()
+{
+    $out = array();
+    $ret = array();
+    exec('/bin/df -P /', $out);
+    # Filesystem Size  Used Avail Use% Mount
+    for ($i=0; $i<count($out); $i++) {
+        if ($i == 0) {
+            continue;
+        }
+        $tmp = explode(" ", preg_replace('/\s+/', ' ', $out[$i]));
+        // skip fs ($tmp[0]) and perc_used ($tmp[4])
+        if ($tmp[5] == "/") {
+          $tmp[5] = "root";
+        }
+        $ret[$tmp[5]] = array('total' => intval($tmp[1]), 'used' => intval($tmp[2]), 'free' => intval($tmp[3]));
+    }
+    return $ret; 
+}
+
+function readRaid()
+{
+    $ret = array();
+    $out = array();
+    exec('/usr/libexec/nethserver/raid-status', $out, $exit_code);
+    $out = json_decode($out[0], true);
+    $records = count($out["results"]);
+    
+    if ($records > 0) {
+      if ($exit_code == 0) {
+        $ret['status'] = 'ok';
+      } else {
+        $ret['status'] = 'error';
+      }
+    } else {
+      $ret['status'] = false;
```

Comment:
I know it's just a matter of taste... I prefer `NULL` to represent a missing information. Here to represent a missing RAID device I'd use NULL instead of FALSE.

```suggestion
      $ret['status'] = NULL;
```

Reply:
I love philosophical discussions ^-^
I will change it in a minute :)